pkg/apply/applier.go

```diff
@@ -342,22 +342,21 @@ func handleError(eventChannel chan event.Event, err error) {
 	}
 }
 
-var coreV1Namespace = v1.SchemeGroupVersion.WithKind("Namespace")
-
-// validateNamespace returns true if all the objects in the passed
-// infos parameter have the same namespace; false otherwise. Ignores
-// cluster-scoped resources.
-func validateNamespace(inv *resource.Info, infos []*resource.Info) error {
+// inventoryNamespaceInSet returns the the namespace the passed inventory
+// object will be applied to, or nil if this namespace object does not exist
+// in the passed slice "infos" or the inventory object is cluster-scoped.
+func inventoryNamespaceInSet(inv *resource.Info, infos []*resource.Info) *resource.Info {
```

Comment:
Not something we should address here, but we have at least two different types now that requires "special" handling and we end up looking for them among the slice of infos. I think we should probably create a new type that wraps the infos slice so we can put this functionality there.

Reply:
Ack.